Makefile

```diff
@@ -159,25 +159,29 @@ $(SCOPE_UI_TOOLCHAIN_UPTODATE): client/yarn.lock $(SCOPE_UI_BUILD_UPTODATE)
 	touch $(SCOPE_UI_TOOLCHAIN_UPTODATE)
 
 client/build/index.html: $(shell find client/app -type f) $(SCOPE_UI_TOOLCHAIN_UPTODATE)
-	mkdir -p client/build
 	if test "true" != "$(SCOPE_SKIP_UI_ASSETS)"; then \
+		rm -rf client/build; \
 		$(SUDO) docker run $(RM) $(RUN_FLAGS) \
 			-v $(shell pwd)/.cache:/home/weave/scope/.cache \
 			-v $(shell pwd)/client:/home/weave/scope/client \
 			-v $(shell pwd)/$(SCOPE_UI_TOOLCHAIN):/home/weave/scope/client/node_modules \
 			-w /home/weave/scope/client \
-			$(SCOPE_UI_BUILD_IMAGE) yarn run build; \
+			$(SCOPE_UI_BUILD_IMAGE) yarn run build-docker; \
+		mkdir -p client/build; \
+		cp -r client/build-docker/* client/build; \
 	fi
 
 client/build-external/index.html: $(shell find client/app -type f) $(SCOPE_UI_TOOLCHAIN_UPTODATE)
-	mkdir -p client/build-external
 	if test "true" != "$(SCOPE_SKIP_UI_ASSETS)"; then \
+		rm -rf client/build-external; \
```

Comment:
Would be nice to re-use `yarn clean-build-external` here if possible..

Reply:
Yup I was thinking like that. Will add. Thanks.